community/sway/etc/skel/.config/waybar/style.css

```diff
@@ -132,6 +132,8 @@
 
 #custom-menu {
     background-color: @theme_selected_bg_color;
+    padding-left: 10px;
```

Comment:
completely unrelated: with the nerd font, the manjaro logo needed some custom padding around.